src/backend/tests/change-requests.test.ts

```diff
@@ -102,6 +102,14 @@ describe('Change Requests', () => {
       expect(prisma.change_Request.findUnique).toHaveBeenCalledTimes(1);
     });
 
+    test('the associated scope of cr does not exist', async () => {
+      jest.spyOn(prisma.change_Request, 'findUnique').mockResolvedValueOnce(changeRequest);
+      await expect(() =>
+        ChangeRequestsService.reviewChangeRequest(superman, crId, reviewNotes, accepted, null)
```

Comment:
you are testing the addProposedSolution service function not the review change request function